engine/src/main/java/org/hibernate/validator/messageinterpolation/ResourceBundleMessageInterpolator.java

```diff
@@ -115,8 +117,10 @@ public ResourceBundleMessageInterpolator(ResourceBundleLocator userResourceBundl
 			Set<Locale> localesToInitialize,
```

Comment:
Looks like you missed this "localesToInitialize", and a few others below. Maybe try a search and replace?